### PR TITLE
Feature/pep8 refactoring code

### PR DESCRIPTION
**Pull request by renejal**
# add pre-commit
# fix of pep8 guidelines

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      - id: check-json
+      - id: name-tests-test
+      - id: no-commit-to-branch
+  - repo: https://github.com/psf/black
+    rev: 23.3.0
+    hooks:
+      - id: black
+        args:
+          - --line-length=79
+  # - repo: https://github.com/pre-commit/mirrors-autopep8
```
Comment (Gaviria9601) [inline .pre-commit-config.yaml]: Deleted commented block.

File: dojo/settings/settings.dist.py [hunk 2]
```diff
@@ -972,256 +1038,255 @@ def saml2_attrib_map_format(dict):
...
-        secret_broker['hostname'],
-        secret_broker['port'],
-        secret_broker['virtualhost'],
-        env('DD_CELERY_BROKER_PARAMS')
-    )
-else:
-    CELERY_BROKER_URL = env('DD_CELERY_BROKER_URL') \
-        if len(env('DD_CELERY_BROKER_URL')) > 0 else generate_url(
-        env('DD_CELERY_BROKER_SCHEME'),
+CELERY_BROKER_URL = (
```
Comment (Gaviria9601) [inline dojo/settings/settings.dist.py]: The use of secrets manager was eliminated. This should be

**Review by Gaviria9601 — COMMENTED** [review]
Please. Review the comments.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (renejal) [reply .pre-commit-config.yaml]: hook are enabled

File: dojo/settings/settings.dist.py (hunk 2 above)
Comment (renejal) [reply dojo/settings/settings.dist.py]: Error in mergue corrected

**Pull request closed without merging**